### PR TITLE
Added/updated Longmoor lashings and Lamentin Losers

### PR DESCRIPTION
**Pull request by TheLocalPub**

### CODE REVIEW

File: worlds/TheLocalPub/LongmoorLashings/Prefabs/Characters/Factions/BLUFOR/US_Army/GreenBerets/Character_US_SF_Officer_P_LandNav.et [hunk 1]
```diff
@@ -0,0 +1,33 @@
+SCR_ChimeraCharacter : "{C0F45FA0C11853D1}Prefabs/Characters/Factions/BLUFOR/US_Army/GreenBerets/Character_US_SF_Officer.et" {
```
Comment (joewhite94) [inline worlds/TheLocalPub/LongmoorLashings/Prefabs/Characters/Factions/BLUFOR/US_Army/GreenBerets/Character_US_SF_Officer_P_LandNav.et:1]: Are these landnav course prefabs used in the mission?

**Comment by TheLocalPub** [comment]
Updated Lemantin losers to actually have QRF spawn, fixed mines not being armed, and turrets not being mountable.
I also added system messages that now work per side.

**Comment by TheLocalPub** [comment]
Updated LongmoorLashings

**Review by Til-Weimann — APPROVED** [review]
Will fix a remaining issue later

**Pull request merged**